client/main.coffee

```diff
@@ -250,13 +256,13 @@ tools =
       ], returnStubValue: true
       console.log 'yo', eventToPoint e
     up: (e) ->
-      return unless pointers[e.pointerId]
+      return unless pointers[e.pointerId] && !restrictTouch e
       undoableOp
         type: 'new'
         obj: Objects.findOne pointers[e.pointerId]
       delete pointers[e.pointerId]
     move: (e) ->
-      return unless pointers[e.pointerId]
+      return unless pointers[e.pointerId] && !restrictTouch e
```

Comment:
Stylistically, I'd split this up into two lines, where the new line is `return if restrictTouch e`

Reply:
Fixed in the new commit!